recipe/meta.yaml

```diff
@@ -1,22 +1,25 @@
 {% set name = "accelerate" %}
-{% set version = "0.21.0" %}
+{% set version = "0.33.0" %}
 
 package:
   name: huggingface_{{ name|lower }}
```

Comment:
Should it be `accelerate` instead of `huggingface_accelerate`,  see the conda-forge recipe https://github.com/conda-forge/accelerate-feedstock/blob/main/recipe/meta.yaml#L6? In that case, the downstream package diffusers-torch requires updating later too.
But I'm not 100% sure because I'm not aware of why this naming takes place.

Reply:
We could add an extra output eventually, to transition to accelerate. Let's keep it that way for now.